.github/workflows/ci-backend.yml

```diff
@@ -821,6 +821,12 @@ jobs:
                     exit 1
                   fi
 
+                  # Check async migrations - must pass for Django Tests to be green
```

Comment:
oh, this is interesting... there are a bunch of hobby install issues open saying it's not working for them and mentioning async migrations